GameLogic.Tests/entities/StandardBoard_Test.cs

```diff
@@ -91,20 +91,20 @@ public void BoardAction()
             var piece = new Piece(Color.White, PieceType.Rook);
 
             // Set piece.
-            board[cell] = piece;
-            Assert.That(piece, Is.EqualTo(board[cell]));
+            board.SetPiece(cell, piece);
+            Assert.That(piece, Is.EqualTo(board.GetPiece(cell)));
             Assert.That(board.GetCellsWithPieces().Count, Is.EqualTo(1));
```

Comment:
Alternative:

`Assert.That(board.GetCellsWithPieces(), Has.Count.EqualTo(1));`

Reply:
Another alternative:

`Assert.That(board.GetCellsWithPieces(), Has.Exactly(1).Items)`

Reply:
Fixed!